ui/src/app/components/runtime-configuration-panel/customize-panel.tsx

```diff
@@ -120,6 +121,8 @@ export const CustomizePanel = ({
   }, [analysisConfig.computeType]);
 
   const disableControls = runtimeExists && !canUpdateRuntime(runtimeStatus);
+  const canChangeZone = !currentRuntime || !gcePersistentDisk;
```

Comment:
this logic is not quite right, but it does achieve the correct result.  I would like to change this, for clarity.  it's a bit complicated so let's chat.

Reply:
Replaced references to canChangeZone with !gcePersistentDisk, because they are equivelant.

Reply:
I don't see that change here.  Missing commit?

Reply:
Whoops, I forgot. I opened a small followup PR. Do you mind taking a look? https://github.com/all-of-us/workbench/pull/8842